examples/classic/distributed/Paxos/Paxos.qnt

```diff
@@ -142,14 +167,13 @@ module Paxos {
   (* greater than b (thereby promising not to vote in ballot b).             *)
   (***************************************************************************/
   action Phase2a(b, v) = all {
-    not(msgs.filter(m => m.tag == "2a")
-            .exists(m => m.bal == b)),
+    not(msgs.exists(m => match m { Msg2a(m2a) => m2a.bal == b | _ => false })),
     Quorum.exists(Q =>
-      val Q1b = msgs.filter(m => m.tag == "1b")
-                    .filter(m => (m.acc.in(Q)) and (m.bal == b))
```

Comment:
Note that this seems to have been an error, because messages tagged "1b" do not have an `acc` field. This may be good evidence of the advantage of the stricter typing discipline of the new implementation of sum-types.

Reply:
I'm not sure what motivated the naming. The names in this spec are mostly cryptic and inscrutable to me :D I suspect it would be a good idea to do a pass to give less cryptic names at some point, but I'll not tackle name fixes here.